cassovary-benchmarks/src/main/scala/com/twitter/cassovary/PerformanceBenchmark.scala

```diff
@@ -83,6 +83,9 @@ object PerformanceBenchmark extends App with GzipGraphDownloader {
   val helpFlag = flags("h", false, "Print usage")
   val globalPRFlag = flags("globalpr", false, "run global pagerank benchmark")
   val pprFlag = flags("ppr", false, "run personalized pagerank benchmark")
+  val idcFlag = flags("idc", false, "run in-degree centrality benchmark")
```

Comment:
In the interest of not bloating the command args, I'll suggest doing this hierarchically, i.e., "-c" could denote that we should run a centrality benchmark and the followon value could be one of "indegree" or "outdegree" or "closeness" or "all".